internal/params/config.go

```diff
@@ -34,9 +34,9 @@ var (
 	// TestnetChainConfig contains the chain parameters to run a node on the harmony test network.
 	TestnetChainConfig = &ChainConfig{
 		ChainID:        TestnetChainID,
-		CrossTxEpoch:   big.NewInt(1),
-		CrossLinkEpoch: big.NewInt(2),
-		StakingEpoch:   big.NewInt(3),
+		CrossTxEpoch:   big.NewInt(0),
+		CrossLinkEpoch: EpochTBD,
```

Comment:
why this is TBD? don't we need it to enable the crossshard?

Reply:
Cross-shard is enabled from epoch 0 (`CrossTxEpoch`); `CrossLinkEpoch` is for cross-links in beaconchain, which CX does not use yet.